app/views/users/users/_form.html.slim

```diff
@@ -9,7 +9,8 @@
     .col-md-6= f.input :birth_name, placeholder: 'Nom de naissance'
     .col-md-6= f.input :birth_date, as: :date, html5: true
   = f.input :phone_number, as: :tel
-  = f.input :address, input_html: { class: "places-js-container" }
+  - address_value = defined?(rdv_wizard) && user.address.nil? ? rdv_wizard[:where] : user.address
```

Comment:
please use `local_assigns[:rdv_wizard].present?` rather than `defined?` I  think it's less dangerous as it's less eager to reach to global variables